components/StyledLinkSvgContainer/index.js

```diff
@@ -20,12 +20,12 @@ const StyledSpan = styled.span`
   justify-content: center;
 `;
 
-export default function StyledLinkSvgContainer({ dynamicId, svg }) {
+export default function StyledLinkSvgContainer({ id, svg }) {
   let href = "";
   if (svg === infoSVG) {
-    href = `/contacts/${dynamicId}`;
+    href = `/contacts/${id}`;
   } else {
-    href = `/contacts-call/${dynamicId}`;
+    href = `/contacts-call/${id}`;
   }
```

Comment:
You could also save this value in a `const`:

```js
const link = svg ==== infoSVG ? `/contacts/${id}` :  `/contacts-call/${id}`
```

Reply:
Thank you for your thoughtful comments.

This was already suggested to me by another coach but I know that in the future I will have to add more buttons so the ternary operator will not work for me.  I need to choose for several buttons and not just 2.   Please allow me to leave it this way, as it will make my future work easier.

Reply:
Ok this is fine for me then. :)